### ISSUE TITLE
Accessibility testing v3.0.11: Saved screen labels 

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#567]
iOS:

On iOS devices, the screen flow does not recognize the bookmark icon. When it does, the label should be changed to unsave button as well. English: Unsave, French: Enregistrer

Android

On Android devices, instead of labelling the bookmark as save item, it should be labelled as Unsave item.

**Comment by weglmar** [comment]
@ForumG-SC Hi Forum, would it be possible to see an example of what it would look like? Thank you!
